test/ui/starter-select.test.ts

```diff
@@ -190,15 +190,15 @@ describe("UI - Starter select", () => {
 
     await new Promise<void>((resolve) => {
       game.onNextPrompt("SelectStarterPhase", UiMode.STARTER_SELECT, () => {
-        const handler = game.scene.ui.getHandler() as StarterSelectUiHandler;
+        const handler = game.scene.ui.getCurrentHandler() as StarterSelectUiHandler;
         handler.processInput(Button.SUBMIT);
       });
       game.onNextPrompt("SelectStarterPhase", UiMode.CONFIRM, () => {
-        const handler = game.scene.ui.getHandler() as StarterSelectUiHandler;
+        const handler = game.scene.ui.getCurrentHandler() as StarterSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        const handler = game.scene.ui.getCurrentHandler<StarterSelectUiHandler>();
```